dashboard/src/components/users/EditUserForm/EditUserForm.tsx

```diff
@@ -0,0 +1,464 @@
+              hideEmptyHelperText
+              error={!!errors.displayName}
+              helperText={errors?.displayName?.message}
+              fullWidth
+              autoComplete="off"
+            />
+            <Input
+              {...register('avatarURL')}
+              id="Avatar URL"
+              label="Avatar URL"
+              variant="inline"
+              placeholder="Enter Avatar URL"
+              hideEmptyHelperText
+              error={!!errors.avatarURL}
+              helperText={errors?.avatarURL?.message}
+              fullWidth
+              autoComplete="off"
+            />
+            <Input
+              {...register('email')}
+              id="email"
+              label="Email"
+              variant="inline"
+              placeholder="Enter Email"
+              hideEmptyHelperText
+              error={!!errors.email}
+              helperText={
+                errors.email ? (
+                  errors?.email?.message
+                ) : (
+                  <ControlledCheckbox
+                    id="emailVerified"
+                    name="emailVerified"
+                    label="Verified"
+                  />
+                )
+              }
+              fullWidth
+              autoComplete="off"
+            />
+
+            <div className="grid items-center grid-flow-col grid-cols-8 col-span-1 my-1">
+              <div className="col-span-2 ">
+                <InputLabel as="h3">Password</InputLabel>
+              </div>
+              <Button
+                color="primary"
+                variant="borderless"
+                className="col-span-6 px-2 place-self-start"
+                onClick={handleChangeUserPassword}
+              >
+                Change
+              </Button>
+            </div>
+
+            <Input
+              {...register('phoneNumber')}
+              id="phoneNumber"
+              label="Phone Number"
+              variant="inline"
+              placeholder="Enter Phone Number"
+              error={!!errors.phoneNumber}
+              fullWidth
+              autoComplete="off"
+              helperText={
+                errors.phoneNumber ? (
+                  errors?.phoneNumber?.message
+                ) : (
+                  <ControlledCheckbox
+                    id="phoneNumberVerified"
+                    name="phoneNumberVerified"
+                    label="Verified"
+                    disabled={!form.watch('phoneNumber')}
+                  />
+                )
+              }
+            />
+            <ControlledSelect
+              {...register('locale')}
+              id="locale"
+              variant="inline"
+              label="Locale"
+              slotProps={{ root: { className: 'truncate' } }}
+              fullWidth
+              error={!!errors.locale}
+              helperText={errors?.locale?.message}
+            >
+              <Option value="en">en</Option>
```

Comment:
Shouldn't this list be dynamic?

Reply:
:thinking: Yes, technically this should be dynamic (pull from the `auth_locale_allowed_locales` column of the app), but we should be mindful that as of right now there's no way to edit this column (for example, on the settings page `/settings/authentication`) and most apps just come with `en` even though we also support `fr` @elitan @nunopato @plmercereau

Reply:
FYI: This will also need an update for our back-end permissions (`select`)

Reply:
You should be able to get this information from the remote project's backend, just like roles. It should work the same way, and it should be dynamic.

Reply:
I'll also assume from the front-end that side that is a `AUTH_LOCALE_ALLOWED_LOCALES` a string of comma-separated locales (ref [https://github.com/nhost/hasura-auth/blob/main/docs/environment-variables.md#:~:text=AUTH_LOCALE_ALLOWED_LOCALES](https://github.com/nhost/hasura-auth/blob/main/docs/environment-variables.md#:~:text=AUTH_LOCALE_ALLOWED_LOCALES) @plmercereau)

Reply:
It would be best to read it directly from the remote app's `auth.locales` table. Isn't that how you do it with roles? You read the data from `auth.roles` form the remote app?

If yes, do the same for `auth.locales`. If not, are there any reasons not to read roles from `auth.roles`?